Lib/idlelib/configdialog.py

```diff
@@ -2103,6 +2103,10 @@ def load_general_cfg(self):
 
         # Set additional help sources.
         self.user_helplist = idleConf.GetAllExtraHelpSourcesList()
+        self.set_additional_help_sources()
+
+    def set_additional_help_sources(self):
```

Comment:
IMO this method should also be setting `self.user_helplist = idleConf.GetAllExtraHelpSourcesList()` rather than assuming it is set elsewhere before it is called.